docs/developers/optimistic-oracle-integration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Integrating with the Optimistic Oracle
+sidebar_label: Integrating with the Optimistic Oracle
+---
+
+## "Deposit Box" Example Contract
+
+This example will set up an `OptimisticDepositBox` contract which custodies a user’s ERC-20 token balance.
+
+On a local testnet blockchain, the user will deposit wETH (Wrapped Ether) into the contract and withdraw wETH corresponding to a desired USD amount.
```

Comment:
When I read this, it does not make intuitive sense. So wETH goes in and wETH comes out? Or is it wETH goes in and an ERC-20 valued at $1 comes out?

Reply:
A USD-denominated amount of wETH comes out. I can try to make this clearer!